### PR TITLE
[TWallet] it stops emitting `register` event to prevent TWallet button to render twice

### PR DESCRIPTION
**Pull request by lcs86-dev**
it stops emitting `register` event when creating TWallet to prevent TWallet button to render twice

### CODE REVIEW

**Comment by 0xmaayan** [comment]
Thank you @lcs86-dev for the PR. Please run `pnpm run changeset` (can follow instructions [here](https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/CONTRIBUTING.md#changesets)) so it will get included in the next release

**Comment by lcs86-dev** [comment]
> Thank you @lcs86-dev for the PR. Please run `pnpm run changeset` (can follow instructions [here](https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/CONTRIBUTING.md#changesets)) so it will get included in the next release

I just added changeset. Thank you.

**Comment by 0xmaayan** [comment]
> > Thank you @lcs86-dev for the PR. Please run `pnpm run changeset` (can follow instructions [here](https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/CONTRIBUTING.md#changesets)) so it will get included in the next release
> 
> I just added changeset. Thank you.

Hi @lcs86-dev , thanks for looking at it! I was out last week. Looks like this branch has conflicts that need to be resolved - once it is resolved I can run CI and merge it

**Review by 0xmaayan — APPROVED** [review]

File: packages/wallet-adapter-core/package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "buffer": "^6.0.3",
     "eventemitter3": "^4.0.7",
     "tweetnacl": "^1.0.3",
-    "@atomrigslab/aptos-wallet-adapter": "^0.1.12",
+    "@atomrigslab/aptos-wallet-adapter": "^0.1.13"
```
Comment (0xmaayan) [inline packages/wallet-adapter-core/package.json]: build is failing 😄 
```suggestion
    "@atomrigslab/aptos-wallet-adapter": "^0.1.13",
```

Comment (lcs86-dev) [reply packages/wallet-adapter-core/package.json]: @0xmaayan sorry, i just fixed it 😱

**Pull request merged**